vendor/vendor.json

```diff
@@ -71,6 +71,12 @@
 			"revisionTime": "2016-01-10T10:55:54Z"
 		},
 		{
+			"checksumSHA1": "uPCqYCT45y5/tdWgMJgShUTJ+y0=",
+			"path": "github.com/shibukawa/configdir",
```

Comment:
neat!

If you can find time this week, do you thin you could update `config/serialization.go` to use github.com/shibukawa/configdir, too (in a separate PR)?  If not, please do open a Github issue on the topic so we don't forget.